demos/codelab/completed/finish/scripts/sound_blocks.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2017 Google Inc.
+ * https://developers.google.com/blockly/
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ Blockly.Blocks['play_sound'] = {
```

Comment:
My concern is that the canonical documentation never mentions defineBlocksWithJsonArray: https://developers.google.com/blockly/guides/create-custom-blocks/define-blocks. If we make the codelab too disconnected from the main documentation, it will be confusing for developers. Maybe we should update the main docs and only then change the codelab? It is an accompanying resource to the main doc.
Wdyt?

Reply:
@devnook good point.  We should update the main docs to mention defineBlocksWithJsonArray, probably here: https://developers.google.com/blockly/guides/configure/web/custom-blocks and here: https://developers.google.com/blockly/guides/create-custom-blocks/define-blocks

Fortunately updating and repushing the docs is pretty fast for us, so I would lean toward using defineBlocksWithJsonArray here and updating the other docs at the same time or as a fast follow.

Reply:
Filed as #1333 so we don't forget.